src/Framework/TestCase.php

```diff
@@ -1676,22 +1676,20 @@ protected function getMockForTrait($traitName, array $arguments = [], $mockClass
      * @param bool   $callOriginalConstructor
      * @param bool   $callOriginalClone
      * @param bool   $callAutoload
-     * @param bool   $cloneArguments
      *
      * @return object
      *
      * @throws Exception
      */
-    protected function getObjectForTrait($traitName, array $arguments = [], $traitClassName = '', $callOriginalConstructor = true, $callOriginalClone = true, $callAutoload = true, $cloneArguments = false)
+    protected function getObjectForTrait($traitName, array $arguments = [], $traitClassName = '', $callOriginalConstructor = true, $callOriginalClone = true, $callAutoload = true)
     {
         return $this->getMockObjectGenerator()->getObjectForTrait(
```

Comment:
For reference, see [`PHPUnit_Framework_MockObject_Generator::getObjectForTrait()`](https://github.com/sebastianbergmann/phpunit-mock-objects/blob/master/src/Generator.php#L405-L444), which accepts 6 parameters only, so `$cloneArguments` is never used.